suricata/update/main.py

```diff
@@ -692,7 +692,14 @@ def build_rule_map(rules):
         if rule.id not in rulemap:
             rulemap[rule.id] = rule
         else:
+            if rule["rev"] == rulemap[rule.id]["rev"]:
+                logger.warning(
+                    "Found duplicate rule SID {}, keeping existing rule."
+                    .format(rule.sid))
             if rule["rev"] > rulemap[rule.id]["rev"]:
+                logger.warning(
+                    "Found duplicate rule SID {}, "
+                    "keeping one with higher revision.".format(rule.sid))
```

Comment:
should this be something like 'keeping *the* one with the *highest* revision." ?

Reply:
Maybe instead of ```the one``` it reads better to have ```the rule``` ?

Reply:
In the first case, I think I'd like a little more information:
```
Found duplicate rule SID {} with same revision, keeping first rule seen.
```

And like Victor said for the second case:
```
Found duplicate rule SID {}, keeping rule with greater revision."
```

Reply:
Thanks. This sounds much better. Addressing the changes in a new PR.